docs/modules/tile-converter/cli-reference/tile-converter.md

```diff
@@ -115,29 +117,29 @@ npx tile-converter --input-type 3DTILES --tileset ..... --metadata-class bridges
 
 ## Quiet mode
 
-Use `--quet` option to avoid user prompts during conversion. In some cases, tile-converter stops and waits for an input from a user, for example, asks weither resume conversion or start a new one.
+Use `--quiet` option to avoid user prompts during conversion. In some cases, tile-converter stops and waits for an input from a user, for example, it can ask whether to resume conversion or start anew.
```

Comment:
```suggestion
Use `--quiet` option to avoid user prompts during conversion. In some cases, tile-converter stops and waits for an input from a user, for example, it can ask whether to resume conversion or start a new.
```

Reply:
```suggestion
Use `--quiet` option to avoid user prompts during conversion. In some cases, tile-converter stops and waits for an input from a user, for example, it can ask whether to resume conversion or start a new conversion.
```